phoenixRest/views/agenda/__init__.py

```diff
@@ -52,13 +52,48 @@ def create_agenda_entry(context, request):
         return {
             "error": "Event not found"
         }
+
+
     parsed_time = datetime.fromtimestamp(int(request.json_body['time']))
+    parsed_newTime = datetime.fromtimestamp(int(request.json_body['newTime']))
 
     entry = AgendaEntry(title=request.json_body['title'],
                         description=request.json_body['description'],
                         time=parsed_time, 
-                        event=event)
+                        event=event,
+                        sticky=sticky,
+                        location=location,
+                        newTime=parsed_newTime,
+                        newLocation=newLocation,
```

Comment:
Where is this variable defined?
Is the value of `newLocation` supposed to be controlled by the user when the agenda entry is initially created, or only when it is updated? To me it seems like `newLocation` is used to store the fact that an agenda entry has been modified.

Reply:
The purpose of this variable is to let the user set a planned location, and a newLocation (now deviating_location) if the location has been changed because ex. overlapping. Both variables can be changed afterwards.

In the latest commit, the variable has changed name and is no longer required in any of the functions (aka. create_agenda_entry and modify_agenda_entry).